spyder/app/mainwindow.py

```diff
@@ -2676,10 +2676,18 @@ def report_issue(self, body=None, title=None, open_webpage=False):
                 url.setQuery(query)
                 QDesktopServices.openUrl(url)
 
-    def report_lsp_error(self, message):
-        """Report a LSP error in the main window"""
-        # TODO: Display error message in a Qt widget
-        pass
+    def report_pyls_error(self, message):
+        """Report a PyLS internal error in our report dialog."""
+        from spyder.widget.reporterror import SpyderErrorDialog
+        error_dialog = SpyderErrorDialog(self, is_report=True)
+        title = _("Internal Python Language Server error")
+        description = _("An error has occurred with the code completion and "
+                        "linting services provided by the Python Language "
+                        "Server. Please submit this report to Github if you "
+                        "want to see this fixed.")
+        error_dialog.set_title_description(title, description)

```

Comment:
I think you misunderstood me. This needs to be

    error_dialog.set_title(title)
    error_dialog.set_description(description)

I'll explain below what these methods do.